### DISCUSSION TITLE
msfs 2024

### DISCUSSION THREAD

**Discussion by blb92** [community/community#147231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, does little nav map will be compatible with msfs 2024 ?

**Comment by ebndev** [comment]
Hi @blb92,

Thanks for being a part of the GitHub Community, we're glad you're here!

I have moved this post to General, however, it looks like you're looking for support from [Little Navmap](https://albar965.github.io/). You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!
